docs/site/decorators/Decorators_openapi.md

```diff
@@ -496,6 +496,187 @@ class MyOtherController {
 }
 ```
 
+### @oas.response
+
+[API document](https://loopback.io/doc/en/lb4/apidocs.openapi-v3.response.html),
```

Comment:
It will be https://loopback.io/doc/en/lb4/apidocs.openapi-v3.oas.response.html.

Reply:
:+1: